provisions/roles/jenkins/common/tasks/main.yml

```diff
@@ -8,6 +8,16 @@
   - jenkins/slave
   - config
 
+    #- name: Ensure pip is installed.
+    #  yum: name=python-pip state=present
+    #  tags:
+    #    - jenkins
+    #    - jenkins/master
+    #    - jenkins/slave
+
+      #- name: Ensure django is installed
+      #  shell: "pip install django"
```

Comment:
Are we sure that not installing django on jenkins master & slave is OK?

Reply:
I actually found a task that installs django on one node some time back, so thanks i will remove this and update the task for more nodes.